lib/MusicBrainz/Server/Edit/Series/Edit.pm

```diff
@@ -82,6 +89,8 @@ sub build_display_data {
 around allow_auto_edit => sub {
     my ($orig, $self, @args) = @_;
 
+    return 1 if $self->can_amend($self->series_id);
```

Comment:
Will that allow people to change the series type if they've already added items of a different type?